frontend/src/lib/primevue/useToast.ts

```diff
@@ -1,14 +1,38 @@
+import type { ToastMessageOptions } from 'primevue/toast';
 import { useToast as useToastPrimevue } from 'primevue/usetoast';
 
-import type { ToastMessageOptions } from 'primevue/toast';
+import { ToastTimeout } from '@/utils/constants';
 
 export const useToast = () => {
   const toast = useToastPrimevue();
 
-  const add = (options: ToastMessageOptions = {}) => {
-    const { severity = 'error', summary = 'Unable to load data.', detail = '', life = 5000 } = options;
+  const error = (title: string, msg: string, options: ToastMessageOptions = {}) => {
+    const {
+      severity = 'error',
+      summary = 'Error: ' + title,
+      detail = msg,
+      life = ToastTimeout.ERROR
+    } = options;
+    toast.add({ severity: severity, summary: summary, detail: detail, life: life });
+  };
+  const info = (title: string, msg: string, options: ToastMessageOptions = {}) => {
+    const {
+      severity = 'info',
+      summary = 'Info: ' + title,
+      detail = msg,
+      life = ToastTimeout.SUCCESS
+    } = options;
+    toast.add({ severity: severity, summary: summary, detail: detail, life: life });
+  };
+  const success = (title: string, msg: string, options: ToastMessageOptions = {}) => {
+    const {
+      severity = 'error',
+      summary = 'Sucess: ' + title,
+      detail = msg,
+      life = ToastTimeout.SUCCESS
+    } = options;
```

Comment:
If we're going to be defining alert types, we should make sure there's also a warning default type as well.